style/style.css

```diff
@@ -0,0 +1,156 @@
+* {
+  margin: 0;
+  padding: 0;
+  font-family: sans-serif;
+  box-sizing: border-box;
+}
+body {
+  background-color: grey;
+}
+header {
+  background: black url(../images/header-image.jpg)100% 100%;
+
+}
+header img{
+  margin-left: 2.5rem;
+}
+.dog-adoption-agency {
+  color: green;
+  font-family: Arial;
+  font-size:1.9rem;
+  margin-left:4.375rem;
+  margin-top: -0.5625rem;
+}
+.heading-right {
+  position: absolute;
+    right: 4.6875rem;
+    width: 12.5rem;
+    margin-top: -7%;
+
+}
+.hello {
+   width:30%;
+ }
+.price{
+  color: white;
+  margin: 15%;
+}
+.checkout-button  a{
+  color: white;
+  text-decoration: none;
+  font-weight: bold;
+
+}
+.checkout-button {
+  background-color: green;
+  width: 12rem;
+  text-align: center;
+  padding: 0.625rem;
+  margin-left: -4rem;
+  margin-top: -1.3rem;
+}
+.top-nav {
+    background-color: #f2f2f2;
+
+}
+ ul {
+
+  overflow: hidden;
+  list-style-type: none;
+  margin-left: 2.5rem;
+ }
+  li {
+  float: left;
+
+ }
+li  a {
+  display: block;
+  text-align: center;
+  padding: 14px 16px;
```

Comment:
Not an issue for now, but make sure you start using % or em/rem units moving forward instead of px. px are not a responsive unit.